src/scrabbleScore.js

```diff
@@ -25,13 +25,31 @@ let SCRABBLE_TILE_SCORES = {
 
 function scrabbleScore(word) {
   // This is your job. :)
+  let adjustedWord = word.toLowerCase();
+  let counter = 0;
+  let sum = 0;
+  let keys = Object.keys(SCRABBLE_TILE_SCORES);
+  let value = Object.values(SCRABBLE_TILE_SCORES);
+  while (counter !== adjustedWord.length) {
```

Comment:
I don't understand the triply-nested loop here. Iterate over each character and add `SCRABBLE_TILE_SCORES[letter]` to the running total as you go. One loop, one iteration per character.